system-variables.md

```diff
@@ -1475,8 +1665,19 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
 - Type: Integer
 - Default value: `0`
 - Range: `[0, 2147483647]`
+
+<CustomContent platform="tidb">
+
 - This variable is used to adjust the maximum days of logger on the current TiDB instance. Its value defaults to the value of the [`max-days`](/tidb-configuration-file.md#max-days) configuration in the configuration file. Changing the variable value only affects the current TiDB instance. After TiDB is restarted, the variable value is reset and the configuration value is not affected.
 
+</CustomContent>
+
+<CustomContent platform="tidb-cloud">
+
+- This variable is used to adjust the maximum days of logger on the current TiDB instance. Changing the variable value only affects the current TiDB instance. After TiDB is restarted, the variable value is reset and the configuration value is not affected.
```

Comment:
A little bit confused. Please confirm whether I understand it right.
```suggestion
- This variable is used to adjust the maximum days of logger on the current TiDB instance. Changing the variable value only affects the current TiDB instance. After TiDB is restarted, the variable value is reset and the original configuration value is not affected.
```

Reply:
The configuration value here means "the value of the [`max-days`](/tidb-configuration-file.md#max-days) configuration in the configuration file". To avoid confusion, let me remove those explanationson for TiDB Cloud because this variable is not applicable to TiDB Cloud.